webpack.config.js

```diff
@@ -17,15 +17,18 @@ module.exports = {
     loaders: [
       {
         test: /\.js$/,
-        loaders:  ['babel-loader']
+        loaders:  ['babel-loader'],
+        include: __dirname + '/www_src'
       },
       {
         test: /\.jsx$/,
-        loaders:  ['babel-loader', 'jsx-loader']
+        loaders:  ['babel-loader', 'jsx-loader'],
+        include: __dirname + '/www_src'
       },
       {
         test: /\.json$/,
-        loader: 'json-loader'
+        loader: 'json-loader',
+        include: [__dirname + '/www_src',  __dirname + '/node_modules/color']
```

Comment:
perhaps I misunderstand what's going on here, then - what does this step do that requires it to reach into the dependency directory? (because that dir should for pretty much all non-require purposes be a black box)